packages/role-governance/src/chain-constants.ts

```diff
@@ -0,0 +1,19 @@
+export const VOLTA_CHAIN_ID = 73799;
```

Comment:
This also defined in `on-chain-enrollment` packages. I think `role-governance` is more appropriate place for them

Reply:
I guess we can have in both, people consuming these packages may need these constants maybe some of it.

Reply:
Or maybe we put in just once and reference it in the other package.